1.10/installing/custom/node-cluster-health-check.md

```diff
@@ -0,0 +1,142 @@
+---
+post_title: Node and Cluster Health Checks
+menu_order: 1000
+---
+
+Node and cluster health checks provide information about your cluster, including available ports, Mesos agent status, and IP detect script validation. The checks are defined during installation in the config.yaml file and can be run by connecting to individual DC/OS nodes via SSH. 
+
+## Cluster Checks
+Cluster checks report the health status of the entire DC/OS cluster. Cluster checks are available across your cluster on all nodes. Common cluster checks include: 
+
+- Is the Mesos leader is present? 
+- Does ZooKeeper have quorum?
+- Can a framework register with Mesos and run a task?
+
+You can discover which cluster checks have been defined by SSHing to your cluster node and running this command: `dcos-diagnostics check cluster --list`.
```

Comment:
Should prepend these commands with `/opt/mesosphere/bin/dcos-shell ` so people don't get confused if their `PATH` doesn't already include `/opt/mesosphere/bin/`.